### PR TITLE
Update robots.txt

### PR DESCRIPTION
**Pull request by alphaCTzo7G**

### CODE REVIEW

**Comment by philippbayer** [comment]
How come I've never seen this PR?!?!

File: public/robots.txt [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
 # Disallow: /
+
+User-agent: SemrushBot
+Disallow: /
+
+User-agent: AhrefsBot
+Disallow: /
+
+User-agent: *
+Disallow: /snps
```
Comment (tsujigiri) [inline public/robots.txt]: Should we keep `/snps` as it contains information people potentially search for?

If anyone has the time, we could try to figure out how much search traffic comes from searches for SNPs. :wink:

Comment (philippbayer) [reply public/robots.txt]: Yes, let's keep `/snps`

**Comment by philippbayer** [comment]
Thank you! See #440 for an update, will merge over there

**Pull request closed without merging**